### PR TITLE
Get proper resource url based on permissions

### PR DESCRIPTION
**Pull request by akucharska**
**Description**

Changes proposed in this pull request:

- Get proper resource url based on permissions

**Related issue(s)**

### CODE REVIEW

File: src/components/Extensibility/helpers/jsonataWrapper.ts [hunk 1]
```diff
@@ -33,5 +36,16 @@ export function jsonataWrapper(expression) {
     return getReadableTimestamp(timestamp);
   });
 
+  exp.registerFunction('canI', (resourceGroupAndVersion, resourceType) => {
```
Comment (qbalukom) [inline src/components/Extensibility/helpers/jsonataWrapper.ts:39]: shouldn't this function be documented?

**Review by qbalukom — CHANGES_REQUESTED** [review]

**Review by qbalukom — DISMISSED** [review]

File: docs/extensibility/jsonata.md [hunk 2]
```diff
@@ -77,6 +78,46 @@ Whenever data sources are provided, they are available as corresponding variable
 
 ## Preset functions
 
+### canI(resourceGroupAndVersion, resourceKind)
+
+You can use this function to determine if user has access rights for listing specified resource.
```
Comment (IwonaLanger) [inline docs/extensibility/jsonata.md:83]:
```suggestion
You can use this function to determine if a user has access rights for listing a specified resource.
```

File: docs/extensibility/jsonata.md [hunk 3]
```diff
@@ -77,6 +78,46 @@ Whenever data sources are provided, they are available as corresponding variable
...
+
+#### Function parameters
+
+- **resourceGroupAndVersion** - first part of a resource url following the pattern: `${resource group}/${resource version}`.
```
Comment (IwonaLanger) [inline docs/extensibility/jsonata.md:87]:
```suggestion
- **resourceGroupAndVersion** - the first part of a resource URL following the pattern: `${resource group}/${resource version}`.
```

File: docs/extensibility/jsonata.md [hunk 4]
```diff
@@ -77,6 +78,46 @@ Whenever data sources are provided, they are available as corresponding variable
...
+You can use this function to sort two strings alphabetically.
+
+#### Function parameters
+
+- **first** - first string to compare.
+- **second** - second string to compare.
+
+#### Example
+
+Example from the [ResourceList widget](display-section.md#resourcelist).
```
Comment (IwonaLanger) [inline docs/extensibility/jsonata.md:109]:
```suggestion
Here is an example from the [ResourceList widget](display-section.md#resourcelist):
```

**Review by IwonaLanger — APPROVED** [review]

**Pull request merged**